helix-view/src/editor.rs

```diff
@@ -110,7 +110,7 @@ pub struct Config {
     /// Automatic auto-completion, automatically pop up without user trigger. Defaults to true.
     pub auto_completion: bool,
     /// Time in milliseconds since last keypress before idle timers trigger. Used for autocompletion, set to 0 for instant. Defaults to 400ms.
-    #[serde(skip_serializing, deserialize_with = "deserialize_duration_millis")]
```

Comment:
I removed this to allow experimenting with the value at runtime, was there any particular reason it was skipping serialization ?

Reply:
I think this is because I had to declare a custom deserializer and I wasn't sure if the default serializer would produce the correct value to roundtrip `x = de(ser(x))`

Reply:
Implemented serialize for it in fdc4e372 just in case.